src/js/modules/competing/views/subscriptions.item.viewparkedmessages.tpl.html

```diff
@@ -0,0 +1,43 @@
+<header class="page-header">
+	<h2 class="page-title">View Parked Messages - {{streamId}}::{{groupName}}</h2>
+	<ul class="page-nav">
+		<li class="page-nav__item">
+			<a ui-sref="^.^.list">Back</a>
+		</li>
+	</ul>
+	
+</header>
+<ul style="list-style-type: none; padding:0 0 0.75rem 0 !important">
+	<li class="page-nav__item"> 
+		<a ng-click="pageFirst(entries)" >Poll</a> 
```

Comment:
Should this not be called "First" instead of "Poll">
```suggestion
		<a ng-click="pageFirst(entries)" >First</a> 
```

Reply:
Thanks, done